ci-deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o nounset -o errexit
+set -x
+
+TMP_DCOS_TERRAFORM=$(mktemp -d)
+git clone https://github.com/bernadinm/dcos-terraform.git $TMP_DCOS_TERRAFORM
```

Comment:
please use either the convenience wrapper or the examples repo. Later would probably the best bet.

Reply:
Yes, the convenience wrapper is used. It's sourcing the examples here for each cloud https://github.com/bernadinm/dcos-terraform/tree/master/examples